src/main/kotlin/org/rust/lang/core/psi/ext/RsElement.kt

```diff
@@ -93,6 +91,14 @@ abstract class RsElementImpl(node: ASTNode) : ASTWrapperPsiElement(node), RsElem
 
     final override val crateRoot: RsMod?
         get() = (contextualFile as? RsElement)?.crateRoot
+
+    // used only for rust console (RsReplCodeFragment)
+    override fun getIcon(flags: Int): Icon? {
```

Comment:
Also, maybe put this logic into `RsPatBindingImplMixin` or something related to `PatBinding`, but not `PatIdent`?